packages/vscode-extension/src/debug/debugChangesNotification.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LocalEnvManager } from "@microsoft/teamsfx-core";
+import { ExtTelemetry } from "../telemetry/extTelemetry";
+import VsCodeLogInstance from "../commonlib/log";
+import { ext } from "../extensionVariables";
+import { vscodeHelper } from "./depsChecker/vscodeHelper";
+import { VS_CODE_UI } from "../extension";
+import * as StringResources from "../resources/Strings.json";
+import {
+  skipNgrokHelpLink,
+  skipNgrokRetiredNotification,
+  trustDevCertHelpLink,
+  trustDevCertRetiredNotification,
+} from "./constants";
+
+export async function showDebugChangesNotification(): Promise<void> {
```

Comment:
Add `TODO` that we can remove this notification after certain releases.

Reply:
Got it.